evap/staff/tests/test_views.py

```diff
@@ -2255,6 +2255,18 @@ def test_num_queries_is_constant(self):
             with self.assertNumQueries(FuzzyInt(0, self.num_questions)):
                 self.app.get(self.url, user=self.manager)
 
+    def test_published(self):
+        let_user_vote_for_evaluation(self.app, self.student2, self.evaluation)
+        Evaluation.objects.filter(id=self.evaluation.id).update(state=Evaluation.State.PUBLISHED)
+        with run_in_staff_mode(self):
+            self.app.get(self.url, user=self.manager, status=403)
```

Comment:
If possible, I'd add the assertion that before the `update()`, we got a 200 response, so that we can be sure the 403 is caused by what we changed here (same for the other tests).

Reply:
That makes sense. I added in the extra assertions :+1: